admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -1050,3 +1050,19 @@
     }
   }
 }
+
+#form_step2_specific_price_sp_id_customer-data {
```

Comment:
Why do we have specific style rules for this field?

If we use the UI kit as requested in the Forge issue, we should not have this kind of stuff in this PR.